feat: Add Test Media Capability

noelmiller opened the pull request:
Fixes: https://github.com/ublue-os/isogenerator/issues/23

akdev1l commented:
`working-directory` I think

akdev1l commented:
the selftest is already helpful! I am happy

p5 commented:
> the selftest is already helpful! I am happy

Much better than going back and forth with the experimental repo!

akdev1l reviewed (approved):
LGTM!

p5 commented:
This PR took way too many attempts! I need to brush up on my bash!

noelmiller committed:
feat: Allow users to override secure boot key and password when using container or action

noelmiller committed:
fix: follow redirects

noelmiller committed:
chore: update docs

noelmiller committed:
fix: added comments and removed additional \n

noelmiller committed:
fix: removed defaults

noelmiller committed:
fix: added conditional for adding public key

noelmiller committed:
chore(ci): Added additional test for secure boot

noelmiller committed:
chore(ci): fixed up workflows to match production

noelmiller committed:
fix(ci): added workflow dispatch to test

noelmiller committed:
fix(ci): fixed version and added IMAGE_TAG

noelmiller committed:
chore(ci): changed to bluefin to test both 38 and 39 secure boot

noelmiller committed:
Merge branch 'main' into decouple_secure_boot_key

noelmiller committed:
chore(ci): added required variables to entrypoint

noelmiller committed:
chore(ci): added merge_group and added paths-ignore

noelmiller committed:
chore(ci): updated description

noelmiller committed:
chore(ci): set to Silverblue since we are testing Bluefin

noelmiller committed:
chore(ci): Fixed secure boot key not found error

Received error in anaconda when check was hit, need to set as a soft failure to exit the script early without stopping anaconda installation.

noelmiller committed:
chore: fixed formatting

noelmiller committed:
chore: fixed whitespace

noelmiller committed:
chore(ci): Removed duplicate test

noelmiller committed:
chore: fix whitespace

noelmiller committed:
chore(ci): fixed test and removed upload to Github Artifacts

noelmiller committed:
chore: updated README

noelmiller committed:
feat: Re-enable Test Media Feature

noelmiller committed:
Merge branch 'main' into add_test_media_capability

bsherman reviewed (approved):
LGTM
